src/vm/dllimport.cpp

```diff
@@ -6563,6 +6563,21 @@ class LoadLibErrorTracker
     DWORD   m_priorityOfLastError;
 };  // class LoadLibErrorTracker
 
+static HMODULE LocalLoadLibraryHelperInternal(LPCWSTR name, DWORD flags)
```

Comment:
Oh, you are right. #ifdef FEATURE_PAL should handle that (looks like I misunderstood what it does). I'm undoing this part of the change. Thanks!